templates/intro-rllib/atari_ppo.py

```diff
@@ -0,0 +1,100 @@
+import gymnasium as gym
```

Comment:
Is there any way, we can re-use the existing (tuned) example script?
Maybe similar to how the rst docs do it? Include certain blocks of the script, one at a time, into the `ipynb`? This would be nice, then we also won't have to worry about the template being tested, b/c it's already tested on the CI.

Reply:
For any given release for which we want to update, we change the RAY_VERSION in 

```
!RAY_VERSION=2.39.0 # Set this when releasing an image instead of hardcoding it here
!curl -O https://raw.githubusercontent.com/ray-project/ray/refs/heads/releases/${RAY_VERSION}/rllib/tuned_examples/ppo/atari_ppo.py"
```

My idea here: That line goes into some file in a utility folder that we just live in the template.
No harm if users explore and find it. And when we want to update the template (which may happen often), we can use that small script every time to grab the newest version of RLlib's files that we need.

Reply:
Done